pcweb/pages/demos_on_landing_page/dashboard/dashboard.py

```diff
@@ -0,0 +1,185 @@
+import reflex as rx
```

Comment:
We should Cole the top level folder just `landing_demos` or something, keep the names concise (can do in a follow up)